package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "movieapp",
+  "version": "1.0.0",
+  "description": "movie application",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack-dev-server --config webpack.dev.js --env.development",
+    "build": "webpack --config webpack.prod.js --env.production",
+    "test": "jest"
+  },
+  "keywords": [
+    "movie",
+    "mark"
+  ],
+  "author": "Mykhailo",
+  "license": "ISC",
+  "dependencies": {
+    "babel-jest": "^24.5.0",
+    "react-redux": "^6.0.1",
+    "react-router": "^4.3.1",
+    "react-test-renderer": "^16.8.4",
+    "redux": "^4.0.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.3.4",
+    "@babel/plugin-proposal-class-properties": "^7.3.4",
+    "@babel/preset-env": "^7.3.4",
+    "@babel/preset-react": "^7.0.0",
+    "babel-loader": "^8.0.5",
+    "css-loader": "^2.1.1",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.11.2",
+    "enzyme-to-json": "^3.3.5",
+    "html-webpack-plugin": "3.2.0",
+    "jest": "^24.5.0",
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
```

Comment:
`react` and `react-dev` are not dev dependencies

Reply:
fixed